app/models/server_puppet_data.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ServerPuppetData
+  attr_reader :client
+
+  def initialize
+    @client = PuppetdbClient.new(Settings.PUPPETDB_ENDPOINT)
+  end
+
+  def self.run
+    new.load_data
+  end
+
+  def load_data
+    Server.find_each do |server|
```

Comment:
I defer to the style guide: https://github.com/bbatsov/rails-style-guide#find-each